openedx/core/djangoapps/user_authn/views/tests/test_logistration.py

```diff
@@ -1,334 +1,41 @@
+""" Tests for Logistration views. """
 
 from __future__ import absolute_import
 
-import json
-import logging
-import re
 from http.cookies import SimpleCookie
-from unittest import skipUnless
 
 import ddt
 import mock
 from django.conf import settings
 from django.contrib import messages
-from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AnonymousUser
 from django.contrib.messages.middleware import MessageMiddleware
 from django.contrib.sessions.middleware import SessionMiddleware
-from django.core import mail
 from django.core.files.uploadedfile import SimpleUploadedFile
 from django.test import TestCase
 from django.test.client import RequestFactory
 from django.test.utils import override_settings
 from django.urls import reverse
 from django.utils.translation import ugettext as _
-from edx_oauth2_provider.tests.factories import AccessTokenFactory, ClientFactory, RefreshTokenFactory
-from oauth2_provider.models import AccessToken as dot_access_token
-from oauth2_provider.models import RefreshToken as dot_refresh_token
-from provider.oauth2.models import AccessToken as dop_access_token
-from provider.oauth2.models import RefreshToken as dop_refresh_token
-from six.moves import range
 from six.moves.urllib.parse import urlencode  # pylint: disable=import-error
-from testfixtures import LogCapture
-from waffle.models import Switch
 
 from course_modes.models import CourseMode
-from openedx.core.djangoapps.oauth_dispatch.tests import factories as dot_factories
 from openedx.core.djangoapps.site_configuration.tests.mixins import SiteMixin
 from openedx.core.djangoapps.theming.tests.test_util import with_comprehensive_theme_context
-from openedx.core.djangoapps.user_api.accounts.api import activate_account
-from openedx.core.djangoapps.user_api.accounts.utils import ENABLE_SECONDARY_EMAIL_FEATURE_SWITCH
-from openedx.core.djangoapps.user_api.errors import UserAPIInternalError
 from openedx.core.djangoapps.user_authn.views.login_form import login_and_registration_form
 from openedx.core.djangolib.js_utils import dump_js_escaped_json
 from openedx.core.djangolib.markup import HTML, Text
-from openedx.core.djangolib.testing.utils import CacheIsolationTestCase, skip_unless_lms
-from student.models import Registration
-from student.tests.factories import AccountRecoveryFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
 from third_party_auth.tests.testutil import ThirdPartyAuthTestMixin, simulate_running_pipeline
 from util.testing import UrlResetMixin
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 
-LOGGER_NAME = 'audit'
-User = get_user_model()  # pylint:disable=invalid-name
-
-FEATURES_WITH_FAILED_PASSWORD_RESET_EMAIL = settings.FEATURES.copy()
-FEATURES_WITH_FAILED_PASSWORD_RESET_EMAIL['ENABLE_PASSWORD_RESET_FAILURE_EMAIL'] = True
-
-
-@skip_unless_lms
-@ddt.ddt
-class UserAccountUpdateTest(CacheIsolationTestCase, UrlResetMixin):
```

Comment:
**Note to Reviewers:** These tests are actually for testing Password Change functionality.  I have moved them to `test_password.py`.  Also, they do not need account-activation functionality for tests to pass.